bower.json

```diff
@@ -14,7 +14,8 @@
     "angular-ui-router": "~0.2.18",
     "bootstrap": "~3.3.6",
     "ng-file-upload": "~12.1.0",
-    "owasp-password-strength-test": "~1.3.0"
+    "owasp-password-strength-test": "~1.3.0",
+    "lodash": "^4.6.1"
```

Comment:
no reason, just never try it :) i have just push this in order to launch the discussion about loadash :)